src/slave/flags.cpp

```diff
@@ -74,8 +74,8 @@ mesos::internal::slave::Flags::Flags()
       "\n"
       "To use JSON, pass a JSON-formatted string or use\n"
       "`--resources=filepath` to specify the resources via a file containing\n"
-      "a JSON-formatted string. `filepath` can be of the form\n"
-      "`file:///path/to/file` or `/path/to/file`.\n"
+      "a JSON-formatted string. `filepath` can only be of the form\n"
+      "`file:///path/to/file`.\n"
```

Comment:
in fact, can you test if `file:///` works on the latest Mesos or Mesos 1.7? By reading code, i doubt that `file:///` is going to work ...

Reply:
Okay... This PR was committed 3 months ago. I'll check the code again.

Reply:
It works on master(1.9.0).